spec/controllers/hub/organizations_controller_spec.rb

```diff
@@ -75,7 +75,7 @@
         it "is not found" do
           expect do
             get :show, params: params
-          end.to raise_error(ActiveRecord::RecordNotFound) # in deployment configs, this would be a 404
+          end.to raise_error(ActionController::RoutingError) # in deployment configs, this would be a 404
```

Comment:
I'd prefer this be RecordNotFound since I think that's more accurate. Alternatively, your custom code could respond with a real 404 status code, and we could adjust this test accordingly.

Not a blocker.